test/cases/parsing/evaluate/index.js

```diff
@@ -70,3 +70,21 @@ it("should allow resourceFragment in context", function () {
 	expect(fn("index")).toEqual("#resourceFragment");
 	expect(fn("returnRF")).toBe("#..");
 });
+
+it("should try to evaluate new RegExp()", function () {
+	function expectAOnly (r) {
+		r.keys().forEach(key => {
+			expect(r(key)).toBe(1);
+		});
+	}
+
+	expectAOnly(
+		require.context("./regexp", false, new RegExp("(?<!filtered)\\.js$", ""))
+	);
+	expectAOnly(
+		require.context("./regexp", false, new RegExp(`(?<!${"filtered"})\\.js$`, "i"))
```

Comment:
```suggestion
		require.context("./regexp", false, new RegExp(`(?<!${"FILTERED"})\\.js$`, "i"))
```

to check if the flag works